pkg/backend/stack.go

```diff
@@ -127,16 +127,53 @@ func GetStackTags() (map[apitype.StackTagName]string, error) {
 			tags[apitype.ProjectDescriptionTag] = *proj.Description
 		}
 
-		if owner, repo, err := gitutil.GetGitHubProjectForOrigin(filepath.Dir(projPath)); err == nil {
-			tags[apitype.GitHubOwnerNameTag] = owner
-			tags[apitype.GitHubRepositoryNameTag] = repo
+		// Add the git metadata to the tags.
+		if err := addGitMetadataToStackTags(tags, projPath); err != nil { // nolint
```

Comment:
When adding `nolint`, it's customary to indicate what specific warning is being ignored. Something like `// nolint: ignoreerr, we intentionally ignore the error.`

But if that is the case, there is a `contract.IgnoreError(err)` (or something similar) that we use to intentionally ignore errors, which IMHO would make this a little cleaner.